### PR TITLE
Feature/feature register pet

### PR DESCRIPTION
**Pull request by marclll**

### CODE REVIEW

File: src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetRestControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            + "  \"recommendedKcal\": 300,\n"
+            + "  \"washFreq\": 3\n"
+            + "}";
+        owner = "Pepe Lotas";
+        name = "ChihuahuaNator";
+        urlBase = "/pet";
+    }
+
+    @Test
+    public void createPetShouldReturnStatusOk() throws Exception {
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetRestControllerTest.java:54]: Codacy Issue found: [Throwing 'Exception' is not allowed.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5240123)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/controller/petmanager/PetRestController.java  100
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5240123&bid=17177520)

File: src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetRestControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        willDoNothing().given(service).createPet(anyString(), anyString(), isA(PetEntity.class));
+        mockMvc.perform(post(urlBase + "/" + owner + "/" + name)
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetRestControllerTest.java:56]: Codacy Issue found: [The String "/" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5240123)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by enrichp — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey, you've got my approval

**Pull request merged**